packages/chains/src/bsc/assets.ts

```diff
@@ -1,258 +1,329 @@
   },
   // Ellipsis
   {
     symbol: assetSymbols["3EPS"],
-    underlying: "0xaF4dE8E872131AE328Ce21D909C74705d3Aaf452",
+    underlying: threeEPS,
     name: "Ellipsis.finance 3EPS (BUSD/USDC/USDT)",
     decimals: 18,
     oracle: OracleTypes.CurveLpTokenPriceOracleNoRegistry,
+    extraDocs: ellipsisDocs("0x160caed03795365f3a589f10c379ffa7d75d4e76", "3EPS", threeEPS),
   },
   {
     symbol: assetSymbols.val3EPS,
-    underlying: "0x5b5bD8913D766D005859CE002533D4838B0Ebbb5",
+    underlying: val3EPS,
     name: "Ellipsis.finance val3EPS (BUSD/USDC/USDT)",
     decimals: 18,
     oracle: OracleTypes.CurveLpTokenPriceOracleNoRegistry,
+    extraDocs: ellipsisDocs("0x19EC9e3F7B21dd27598E7ad5aAe7dC0Db00A806d", "val3EPS", val3EPS),
   },
   {
     symbol: assetSymbols.valdai3EPS,
-    underlying: "0x8087a94FFE6bcF08DC4b4EBB3d28B4Ed75a792aC",
+    underlying: valdai3EPS,
     name: "Ellipsis.finance valdai3EPS (DAI, val3EPS)",
     decimals: 18,
     oracle: OracleTypes.CurveLpTokenPriceOracleNoRegistry,
+    extraDocs: ellipsisDocs("0x245e8bb5427822FB8fd6cE062d8dd853FbcfABF5", "valdai3EPS", valdai3EPS),
   },
 
   {
     symbol: assetSymbols["2brl"],
-    underlying: "0x1B6E11c5DB9B15DE87714eA9934a6c52371CfEA9",
+    underlying: twoBRL,
     name: "Ellipsis.finance 2BRL (BRZ, jBRL)",
     decimals: 18,
     oracle: OracleTypes.CurveLpTokenPriceOracleNoRegistry,
+    extraDocs: ellipsisDocs("0xad51e40D8f255dba1Ad08501D6B1a6ACb7C188f3", "2brl", twoBRL),
   },
   // Bomb
   {
     symbol: assetSymbols.BOMB,
-    underlying: "0x522348779DCb2911539e76A1042aA922F9C47Ee3",
+    underlying: BOMB,
     name: "BOMB",
     decimals: 18,
     oracle: OracleTypes.UniswapTwapPriceOracleV2,
+    extraDocs: defaultDocs("https://bscscan.com", BOMB),
   },
   {
     symbol: assetSymbols.xBOMB,
-    underlying: "0xAf16cB45B8149DA403AF41C63AbFEBFbcd16264b",
+    underlying: xBOMB,
     name: "xBOMB",
     decimals: 18,
     disabled: true,
   },
   {
     symbol: assetSymbols.aBNBc,
-    underlying: "0xE85aFCcDaFBE7F2B096f268e31ccE3da8dA2990A",
+    underlying: aBNBc,
     name: "Ankr BNB Reward Bearing Certificate",
     decimals: 18,
     disabled: true,
+    extraDocs: ankrBNBDocs("aBNBc"),
   },
   {
     symbol: assetSymbols["BTCB-BOMB"],
-    underlying: "0x84392649eb0bC1c1532F2180E58Bae4E1dAbd8D6",
+    underlying: BTCB_BOMB,
     name: "BOMB-BTC PCS LP",
     decimals: 18,
     oracle: OracleTypes.UniswapLpTokenPriceOracle,
+    extraDocs: pcsDocs(BTCB, BOMB, "BOMB-BTC", BTCB_BOMB),
   },
   // Jarvis
   {
     symbol: assetSymbols.jBRL,
-    underlying: "0x316622977073BBC3dF32E7d2A9B3c77596a0a603",
+    underlying: jBRL,
     name: "Jarvis Synthetic Brazilian Real",
     decimals: 18,
     oracle: OracleTypes.ChainlinkPriceOracleV2,
+    extraDocs: `You can acquire this asset on the <a href="https://v1-app.jarvis.exchange/" target="_blank" style="color: #BCAC83; cursor="pointer">Jarvis Network</a> website`,
```

Comment:
We should leave out the `color` here and just style it via global style in the for `<a/>` or even better local styling of `<a/>` children of that tooltip.